r/the-tidyfinance-r-package.qmd

```diff
@@ -0,0 +1,34 @@
+---
+title: The `tidyfinance` R package
+metadata:
+  pagetitle: The `tidyfinance` R package
+  description-meta: How to install and use the companion package to Tidy Finance with R.  
+---
+
+`tidyfinance` is an R package that contains a set of helper functions for empirical research in financial economics, addressing a variety of topics covered in this book. We designed the package to provide easy shortcuts for the applications that we discuss in the book. If you want to inspect the details of the package or propose new features, feel free to visit the package repository on [Github.](https://github.com/tidy-finance/r-tidyfinance)
+
+## Installation
+
+You can install the released version of `tidyfinance` [from CRAN](https://cran.r-project.org/package=tidyfinance) via:
+
+``` r
+install.packages("tidyfinance")
+```
+
+You can install the development version of `tidyfinance` from [GitHub](https://github.com/tidy-finance/r-tidyfinance) via, but keep in mind that it might not be fully tested:
```

Comment:
wording order confusing. Maybe " from Github via (but keep in mind that it might not be fully tested)"

Reply:
Changed it to "You can install the development version of `tidyfinance` from [GitHub](https://github.com/tidy-finance/r-tidyfinance) (which might not be fully tested) via:"